autoload/ale/fixers/fecs.vim

```diff
@@ -0,0 +1,17 @@
+" Author: harttle <yangjvn@126.com>
+" Description: Apply fecs format to a file.
+
```

Comment:
You'll need to set defaults for the options here too, as the fixer might be used by the linter is loaded.

Reply:
Note that I called `ale#handlers#fecs#GetExecutable` instead of  `ale#node#FindExecutable`, and the defaults are already set in the `autoload/ale/handlers/fecs.vim`. So it should be OK I guess? Or do I miss something?

Reply:
You are right, that should work.